src/Api/AdminConsole/Controllers/OrganizationsController.cs

```diff
@@ -245,6 +239,23 @@ public async Task<OrganizationAutoEnrollStatusResponseModel> GetAutoEnrollStatus
         return new OrganizationAutoEnrollStatusResponseModel(organization.Id, data?.AutoEnrollEnabled ?? false);
     }
 
+    [HttpGet("{id}/risks-subscription-failure")]
+    public async Task<OrganizationRisksSubscriptionFailureResponseModel> RisksSubscriptionFailure(string id)
```

Comment:
```suggestion
    public async Task<OrganizationRisksSubscriptionFailureResponseModel> RisksSubscriptionFailure(Guid id)
```
We can change this to a Guid type so we don't have to parse it manually and you can delete `var organizationId = new Guid(id);`

Reply:
@r-tome Good catch thank you. Updated here: https://github.com/bitwarden/server/pull/3515/commits/514086cf9055ae232090e1197d61494f93cecc4e